src/build.py

```diff
@@ -1615,6 +1615,8 @@ def ExecuteEmscriptenTestSuite(name, tests, config, outdir, warn_only=False):
   buildbot.Step('Execute emscripten testsuite (%s)' % name)
   Mkdir(outdir)
   try:
+    proc.check_call([os.path.join(NodeBinDir(), 'npm'), 'install'],
```

Comment:
also. does this need to be wrapped in `Executable()`?

Reply:
Yup .. I think so.